staging/src/k8s.io/kubectl/pkg/cmd/util/factory_client_access.go

```diff
@@ -154,13 +154,8 @@ func (f *factoryImpl) Validator(validationDirective string) (validation.Schema,
 		return validation.NullSchema{}, nil
 	}
 
-	resources, err := f.OpenAPISchema()
```

Comment:
Creation of the `factory` does not download the Open API V2; the Open  API V2 `resources` are only downloaded when they are actually used.